.github/workflows/windows.yml

```
@@ -0,0 +1,100 @@
+---
+name: Foreman Proxy on Windows
+
+on:
+  pull_request:
+  push:
+    branches:
+      - 'develop'
+      - '*-stable'
+
+env:
+  BUNDLE_WITHOUT: "development"
+
+concurrency:
+  group: ${{ github.ref_name }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  bundle-smart-proxy:
+    runs-on: windows-latest
+    defaults:
+      run:
+        shell: pwsh
+    strategy:
+      matrix:
+        rubyversion: ['3.3', '3.4']
+        proxytag: ['3.16.1']
+
+    steps:
+      - name: Determine tag to fetch
+        id: tag
+        run: |
+          $ErrorActionPreference = 'Stop'
+          $repo = 'theforeman/smart-proxy'
+
+          if ('${{ github.event.inputs.version }}') {
+            $tag = '${{ github.event.inputs.version }}'
+          } else {
+            $tag = '${{ matrix.proxytag }}'
+          }
+          "tag=$tag" | Out-File -FilePath $env:GITHUB_OUTPUT -Encoding utf8 -Append
+
+      - name: Set up Ruby (Windows)
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.rubyversion }}
+          bundler-cache: false
+
+      - name: Install Bundler
+        run: |
+          gem install bundler --no-document
+
+      - name: Install rake
+        run: |
+          gem install rake --no-document
```

Why does a regular `bundle install` not work? I'd hope it does
```suggestion
          bundler-cache: true
```